src/rendezvous/MSC3906Rendezvous.ts

```diff
@@ -38,10 +57,21 @@ enum Outcome {
     Unsupported = "unsupported",
 }
 
+enum FailureReason {
```

Comment:
again, let's give this a docstring please. seems to be potential `reason` properties of payloads of type `m.login.failure` ?